client/post-editor/media-modal/secondary-actions.jsx

```diff
@@ -62,6 +64,11 @@ const MediaModalSecondaryActions = React.createClass( {
 		this.props.onChangeView( ModalViews.DETAIL );
 	},
 
+	navigateToPlans() {
+		analytics.ga.recordEvent( 'Media', 'Clicked Plan Storage Button' );
+		page( `/plans/${ this.props.site.slug }` );
```

Comment:
Can we use the new [`getSiteSlug` selector](https://github.com/Automattic/wp-calypso/blob/c322206af0fd40f7d6ef1c81912f895c94606c2f/client/state/sites/selectors.js#L60-L78) here, even if the site object being passed is a decorated one?